README.md

```diff
@@ -73,3 +81,103 @@ optional arguments:
                         associated matrix conditions, a total of Nx2 builds
                         will be identified. (default: 2)
```

Comment:
I'd love to pull this in automatically, but for now the manual step will have to suffice.